hera_corr_cm/hera_corr_cm.py

```diff
@@ -302,7 +302,7 @@ def get_f_status(self):
             'temp': ('temp', float),
             'uptime': ('uptime', int),
             'last_programmed': ('last_programmed', dateutil.parser.parse),
-            'timestamp': ('timestamp', dateutil.parser.parse),
+            'timestamp': ('timestamp', dateutil.parser.parse)
```

Comment:
why?

Reply:
Why remove the unnecessary last comma?  Tidiness I guess.

Reply:
ok, but leaving it means you can add lines later without touching this one, which is why having trailing commas is the black style. I don't need you to change it, but there is a reason for it to be there.